### ISSUE TITLE
support request: how to create a bootenv ?

### ISSUE THREAD

**Issue by eoli3n** [zbm-dev/zfsbootmenu#122]
if I run
```
zfs snapshot zroot/ROOT/void@now
zfs clone zroot/ROOT/void@now zroot/ROOT/void-new
zfs promote zroot/ROOT/void-new
reboot
```
I only see in zfsbootmenu ``void`` dataset.
If i go to snapshot submenu, then ``clone and promote`` to a new name ``void-new2``, it appears in list.

Also, how to chroot a dataset in rw mode from zfsbootmenu ?

**Comment by zdykstra** [comment]
In the most basic case, the cloned snapshot must still have `mountpoint=/` set. If the filesystem doesn't have that configured, ZFSBootMenu doesn't try to mount it and find kernel/initramfs pairs.  

If you'd like to chroot into a boot environment in read/write mode, hit alt-w on the main screen with the boot environment selected. This will re-import the pool in read/write mode and then the subsequent chroot will detect that and allow the filesystem to also be mounted read/write.

**Comment by eoli3n** [comment]
Thanks for your answer.
So the common workflow is to snapshot only, then if needed, clone and promote from zfsbootmenu ?

I don't have any alt-w options, its not documented in alt-h too. If i use it, nothing happens.
![2020-12-20T21:35:00,156603622+01:00](https://user-images.githubusercontent.com/14223657/102723893-cd1f1680-430b-11eb-8bb5-e4f661ca42ed.png)

**Comment by zdykstra** [comment]
The alt-w key combo isn't documented, but it is present. The top line of green text will change to red, and `r/o` becomes `r/w` when it is applied.

**Comment by eoli3n** [comment]
Ok, keyboard issue, my azerty keyboard is configured in qwerty, so ``alt+z`` works.
Would you add ``[ALT+W] mount r-w`` in bottom menu and documentation ?

**Comment by eoli3n** [comment]
``generate-zbm`` with ``hostonly="yes"`` didn't solve it.

**Comment by ahesford** [comment]
In general use, people should avoid importing pools read-write from the boot loader, so I prefer to keep this a secret option.

**Comment by eoli3n** [comment]
As you want, but i think that it could be useful to at least document this, for exemple to be able to use zbm as recovery system to fix current slash dataset.
Is read only chroot useful in any way ?

**Comment by zdykstra** [comment]
A read-only chroot is of some use, but for system recovery read-write is better. I've documented the alt-w toggle in the online help system. For the time being, it will not be documented in the shortcut footer. 

Feel free to update your working copy and let us know if you have any further issues using this software with your keymap - I think we've addressed all of the issues you've uncovered!

We'll be tagging an official release with these fixes in the coming week or two.
